src/github/client/github-client.ts

```diff
@@ -19,17 +20,17 @@ export default class GitHubClient {
 	private readonly axios: AxiosInstance;
 	private readonly appTokenHolder: AppTokenHolder;
 	private readonly installationTokenCache: InstallationTokenCache;
-	private readonly githubInstallationId: number;
-	private logger: Logger;
+	private readonly githubInstallationId: InstallationId;
+	private readonly logger: Logger;
 
 	constructor(
-		githubInstallationId: number,
+		githubInstallationId: InstallationId,
 		logger: Logger,
-		baseURL = "https://api.github.com"
```

Comment:
The url is now encapsulated in the `InstallationId` type, which contains all "coordinates" we need to identify an app installation across cloud or on-premise GitHub instances.